lib/iris/experimental/um.py

```diff
@@ -372,6 +449,18 @@ def _with_source(self):
             if close_required:
                 self.source.close()
 
+    def _read_raw_payload_bytes(self):
+        # Return the raw data payload, as an array of bytes.
+        # This is independent of the packing representation.
+        field = self.lookup_entry
+        with self._with_source():
+            self.source.seek(self.offset)
+            data_size = ((field.lbnrec * 2) - 1) * _WGDOS_SIZE
```

Comment:
Just what I meant by my subsequent comment.
I believe it is basically "wrong", and should probably read ```data_size = field.lbnrec * self.word_size / _WGDOS_SIZE```
Discussed with @rhattersley :  The calculation is a legacy older than the github repo.  As we've done it this way for ages, we can't determine where that came from, and it is not really part of these changes, we deduced it may be safer to leave it alone for now.